examples/askui-nodejs-base-example/package.json

```diff
@@ -7,17 +7,18 @@
         "lint:staged": "lint-staged"
     },
     "dependencies": {
-        "askui": "0.1.7"
+        "askui": "0.1.7",
+        "node-machine-id": "^1.1.12"
```

Comment:
I think this can and should be removed as this is installed as a transitive dependency.